templates/generic-daemonset.yml.j2

```diff
@@ -38,10 +67,12 @@ spec:
             path: {{ mount['host_path'] }}
 {%- endfor %}
       containers:
-{% for container in containers %}
+{%- for container in containers %}
         - name: {{ container['name'] }}
+{%- if container['privileged'] %}
```

Comment:
Please add this flag to all currently prepared daemonsets with privileged containers.